### PR TITLE
SNAP max allotment params

### PR DESCRIPTION
**Pull request by nmrodelo**
Sharding off params from #19

### CODE REVIEW

File: openfisca_us/parameters/benefit/SNAP/max_allotment.yaml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+metadata:
+  unit: currency-USD
+  reference:
+  - title: USDA Cost of Living Adjustments
+    href: https://www.fns.usda.gov/snap/allotment/COLA
+  - title: Supplemental Nutrition Assistance Program (SNAP) Fiscal Year (FY) 2021 Income Eligibility Standards
+    href: https://fns-prod.azureedge.net/sites/default/files/media/file/FY21-Income-Eligibility-Standards.pdf
+  period: month
+
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/SNAP/max_allotment.yaml:148]:
```suggestion

```

File: openfisca_us/parameters/benefit/SNAP/max_allotment.yaml [hunk 2]
```diff
@@ -0,0 +1,148 @@
+description:  Maximum Monthly SNAP Benefit by State and Household Size
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/SNAP/max_allotment.yaml:1]:
```suggestion
description:  Maximum SNAP benefit by geography and household size
```

**Review by MaxGhenis — COMMENTED** [review]
oops, ignore my previous review. i think this is good to go after these changes, but will let nikhil review too

**Review by nikhilwoodruff — APPROVED** [review]
Thanks @nmrodelo!

**Pull request merged**